compoments/report-repair/summary.js

```diff
@@ -0,0 +1,51 @@
+import React, {useEffect, useState} from 'react';
+import SummaryList from '../summaryList';
+import Button from '../button';
+const Summary = ({values, changeLinkUrlValues}) => {
+  let [whatIsRepairProblemLink, setWhatIsRepairProblemLink] = useState(null)
+  let [describeRepairProblemLink, setDescribeRepairProblemLink] = useState(null)
+  useEffect(()=>{
+    setWhatIsRepairProblemLink(changeLinkUrlValues[values.repairLocation])
+    setDescribeRepairProblemLink(changeLinkUrlValues[values.repairProblem])
+    console.log('ADDRESS',values.address)
+  })
+  const personalDetailsSummary = [
+    {pageName:'Repair address', value: values.address+ ',' + values.postcode, link: 'postcode'},
+    {pageName:'Appointment contact number', value: values.contactPersonNumber, link: 'contact-person'},
+  ]
+  const repairDetailsSummary = [
+    { pageName:'Where is the problem?', value: values.repairLocation, link:'repair-location'},
+    { pageName:'What is the problem?', value: values.repairProblem, link: changeLinkUrlValues[values.repairLocation]},
+    { pageName:'What best describes the problem?', values: values.repairProblemBestDescription, link: changeLinkUrlValues[values.repairProblem]},
+    { pageName:'Description', value: values.description?.text, link:'repair-description'}
+  ]
+  const appointmentDetailsSummary = [
+    { pageName:'Date', value: values.availability, link:'repair-availability'},
```

Comment:
The repair availability value displayed ("15th December 2021 8:00am to 12:00pm") does not match the [prototype](https://housingrepairsonlineprototye.herokuapp.com/beta/v4/summary) which includes the word 'between', i.e. 16th November 2021 **between** 8:00am and 12:00pm